robot/impl/resource_manager.go

```diff
@@ -609,7 +611,7 @@ func (manager *resourceManager) completeConfig(
 			} else {
 				verb = "reconfiguring"
 			}
-			manager.logger.CDebugw(ctx, fmt.Sprintf("now %s resource", verb), "resource", resName)
+			manager.logger.CInfow(ctx, fmt.Sprintf("now %s resource", verb), "resource", resName)
```

Comment:
Which call is actually the thing that gets into "Reconfigure"? Is it the `manager.processResource`? There's some logic paths in there that don't result in a reconfigure. Some of them log at debug, e.g: [this model mismatch -> must rebuild](https://github.com/viamrobotics/rdk/blob/22c765a93a9efdc43c3d24612fa661c79d5d3c1f/robot/impl/resource_manager.go#L862-L863).

I don't mind this log being a bit spurious, but I would like the error cases that cause is to not call `Reconfigure` also be logged at Info or higher:
* either directly logging at info (or higher) or
* returning an error that we're confident will be logged at info or higher

Reply:
> Which call is actually the thing that gets into "Reconfigure"? Is it the manager.processResource? There's some logic paths in there that don't result in a reconfigure. Some of them log at debug, e.g: [this model mismatch -> must rebuild](https://github.com/viamrobotics/rdk/blob/22c765a93a9efdc43c3d24612fa661c79d5d3c1f/robot/impl/resource_manager.go#L862-L863).

It is indeed `manager.processResource`, and from what I can tell, it's only a failure to `Reconfigure` (which should result in an eventual logged error) or a forced rebuild that would deviate from actually reconfiguring. Just bumped the "must rebuild" log to info.